Hunting Queries/AzureActivity/Common_Deployed_Resources.txt

```diff
@@ -6,7 +6,7 @@
 // in combination with other signals that show suspicious deployment to evaluate if the resource is one
 // that is commonly being deployed/created or unique.
 //
-// DataSource: #AzureActivity
+// DataSource: #AzureActivity, #Impact
```

Comment:
is the 'Impact intended to be in Tactics?

Reply:
fixed